apps/snitch_core/test/data/schema/rating/rating_option_test.exs

```diff
@@ -0,0 +1,79 @@
+defmodule Snitch.Data.Schema.RatingOptionTest do
+  use ExUnit.Case, async: true
+  use Snitch.DataCase
+
+  alias Snitch.Data.Schema.RatingOption
+
+  @valid_attrs %{
+    code: "1",
+    value: 1,
+    position: 1,
+    rating_id: 105
+  }
+  describe "create_changeset/2 " do
+    test "succeeds" do
+      %{valid?: validity} = RatingOption.create_changeset(%RatingOption{}, @valid_attrs)
+      assert validity
+    end
+
+    test "fails? with missing code" do
```

Comment:
All the `required` related params can be tested in just one test. No need to test failure for every single `key` in schema. 

Just pass empty params object to `create_changeset(%RatingOption{}, %{})` and you will get all the required related errors.